stories/form-message/form-message.stories.js

```diff
@@ -0,0 +1,80 @@
+import '../../dist/form-message.css';
+import '../../dist/form-item.css';
+import '../../dist/form-label.css';
+import '../../dist/input.css';
+import '../../dist/popover.css';
+
+export default {
+    title: 'Components/Forms/FormMessage',
```

Comment:
I think this should be titled Components/Forms//States

Reply:
FormMessage is the component though